React-JS-Projects/Intermediate/Netflix-Clone/README.md

```diff
@@ -0,0 +1,90 @@
+<h1 align='center'><b>💥 Netflix Clone 💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+<!-- enlist all the technologies used to create this project from them (Remove comment using 'ctrl+z' or 'command+z') -->
+
+<div align='center'>
+
+
+  ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+  ![React](https://img.shields.io/badge/react-%2320232a.svg?style=for-the-badge&logo=react&logoColor=%2361DAFB)
+  ![NodeJS](https://img.shields.io/badge/node.js-6DA55F?style=for-the-badge&logo=node.js&logoColor=white)
+ 
+</div>
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Description 📃
+
+<div>
+  <!-- <p>Add Description of the project</p> -->
+    <p>A Netflix clone built with React and integrated with the TMDB API, allowing users to browse and view real-time movie and TV show data with a sleek UI</p>
+</div>
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: How to run it? 🕹️
+
+<!-- Add steps how to run this project -->
+Clone the project
+
+```bash
+  git clone https://github.com/hamzathul/netflix-clone.git
+```
+
+Go to the project directory
+
+```bash
+  cd netflix-clone
+```
+
+Install dependencies
+
+```bash
+  npm install
+```
+
+Start the server
+
+```bash
+  npm run start
+```
+
+- Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+</br>
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+![App Screenshot](https://github.com/user-attachments/assets/04080ffb-11cc-4ee8-a538-6e6820369b12)
+
+## :zap: Working Video 📹
```

Comment:
Remove this heading, if video is not present.

Reply:
Keep it under `Basic` folder

Reply:
Changes have been made